### PR TITLE
add server_time endpoint and update script

### LINKED ISSUE #35: add `GET /server_time` to get server time with unixtime format
fix https://github.com/0chain/block-explorer/issues/286

- add new api `GET /server_time`
- update api document on README.md
- ask @hm90121  to update other docs

### PR DESCRIPTION
**Pull request by Rizary**
fix #35

### CODE REVIEW

**Comment by Rizary** [comment]
Endpoint information in README.md is not accurate anymore.

File: docker.local/bin/start.sh [hunk 1]
```diff
@@ -3,4 +3,4 @@ PWD=`pwd`
 
 echo Starting 0Proxy ...
 
-docker-compose -p 0proxy -f ../docker-compose.yml up
+docker-compose -p 0proxy -f docker.local/docker-compose.yml up
```
Comment (cnlangzi) [inline docker.local/bin/start.sh:6]: please update README.md for this change too.

**Review by cnlangzi — CHANGES_REQUESTED** [review]

**Comment by cnlangzi** [comment]
please add api document README.md like https://github.com/0chain/0proxy#sync . it helps front-end team to check it instead of review code

File: README.md [hunk 2]
```diff
@@ -170,8 +170,14 @@ Output:
...
 Method: `DELETE`
 
+#### Get the current server time in unix format
+Path: `/server_time`
+Method: `GET`
 
-
+Output:
+```
+1637754327
```
Comment (cnlangzi) [inline README.md:179]: it is different form your code. 

{ "time": 1637754327 }

**Review by cnlangzi — CHANGES_REQUESTED** [review]

**Review by cnlangzi — APPROVED** [review]

**Pull request merged**